Lib/test/eintrdata/eintr_tester.py

```diff
@@ -39,13 +39,16 @@ class EINTRBaseTest(unittest.TestCase):
 
     # delay for initial signal delivery
     signal_delay = 0.1
+    # signal handler run time duration
+    signal_duration = 0.05
```

Comment:
Please remove this variable.

Reply:
Regarding both comments: as I've said, for the test case to be valid - we have to make sure the signal handler returns **only after** the timeout was reached. That's exactly the purpose of sleeping. How do you want me to make sure it happened?
I can maybe use a special parameter initialized just before the `select.select()` call which indicates the deadline, and sleep within the signal handler until this deadline is reached. It may be safer for buildbots, but it doesn't save us from sleeping. Honestly, I don't see how you trigger this bug without sleeping inside the signal handler.

Again, just to clarify - the bug only happens if the signal handler fires **before** the deadline, and returns **after**. It follows that the signal handler must take **significant** time.